.infra/application.properties

```diff
@@ -16,4 +16,4 @@ debezium.transforms.Reroute.type=io.debezium.transforms.ByLogicalTableRouter
 debezium.transforms.Reroute.topic.regex=(.*)
 debezium.transforms.Reroute.topic.replacement=%topic%
 debezium.sink.type=pubsub
-quarkus.log.level=TRACE
+quarkus.log.level=ERROR
```

Comment:
Please revert this change. It will affect production and we currently need it to be trace